src/docs/product/issues/performance-issues/n-one-queries.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "N+1 Queries"
+sidebar_order: 10
+description: "Learn more about N+1 queries and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+## N+1 Queries
+
+"N+1 Queries" is a performance problem where the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection has some amount of performance overhead, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when using an Object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
```

Comment:
Trying to avoid the plural/singular issue in the first sentence by making query singular
Attempting to simplify the language in the second sentence - hopefully I haven't changed the meaning. If I have then ignore that change.

```suggestion
An _N+1 query_ is a performance problem in which the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection takes some amount of time, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when you use an object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
```

Reply:
Applied this change, _except_ I kept it as "N+1 Queries are a performance problem" for consistency